bokehjs/package.json

```diff
@@ -35,7 +35,7 @@
     "yargs": "^13.2.2",
     "jsdom": "^9.8.3",
     "websocket": "^1.0.28",
-    "typescript": "3.6.3",
+    "typescript": "^3.7.2",
```

Comment:
Switched to semver range, because we are now guaranteed reproducible installations in CI after PR #9368.